### PR TITLE
[added] command to init template for existing file - e.g. :TemplateInit

### PR DESCRIPTION
**Pull request by toransahu**
The command will insert template to an existing file based on file extension.

### CODE REVIEW

**Comment by toransahu** [comment]
It sounds fantastic. Thanks for your suggestion. I'll make the changes accordingly and will re-submit the  PR.

**Comment by toransahu** [comment]
The command :Template  will work as suggested.
 e.g. :Template (no parameter) will call TLoad() func. 
while  :Template <arg> will call TLoadTemplate() func.

File: doc/template.txt [hunk 1]
```diff
@@ -32,11 +32,18 @@ determine which template is used:
...
 Templates can be manually expanded in the current buffer using the
-|:Template| command. For example, the following will load and expand the
-template for a file matching the pattern `*.c`:
+|:Template| command in two ways. 
+
+For example, the following will load and expand the
+template for a file matching the file extension:
+>
+	:Template
+>
```
Comment (aperezdc) [inline doc/template.txt]: This is fun... I have seen before `<` to close example blocks in the Vim documentation, but what you did here works as well, so let's just keep it :man_shrugging:

**Comment by toransahu** [comment]
Thanks for merging the PR & giving us `vim-template` :)

File: doc/template.txt (hunk 1 above)
Comment (toransahu) [reply doc/template.txt]: Sure. Also, there are some other combinations used in the [docs of the vim](https://github.com/vim/vim/blob/master/runtime/doc/).

**Comment by aperezdc** [comment]
@toransahu You're welcome! Again, thanks for the patch and for your patience making the changes to improve it before merging it. The plugin wouldn't be the same without the contributions from the community :sparkles:

**Pull request merged**